cpu/efm32_common/periph/pwm.c

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2016-2017 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_efm32_common
+ * @ingroup     drivers_periph_pwm
+ * @{
+ *
+ * @file
+ * @brief       Low-level PWM peripheral driver implementation
+ *
+ * @author      Bas Stottelaar <basstottelaar@gmail.com>
+ */
+
+#include "cpu.h"
+
+#include "periph_conf.h"
+#include "periph/gpio.h"
+#include "periph/pwm.h"
+
+#include "em_cmu.h"
+#include "em_timer.h"
+#include "em_timer_utils.h"
+
+/* guard file in case no PWM device was specified */
+#ifdef PWM_NUMOF
```

Comment:
no need anymore, as this file is only build if a pwm feature is defined.

--> please check also all other periph implementation files for this.

Reply:
Done.